### ISSUE TITLE
Use woff2 and woff format fonts in example

### ISSUE THREAD

**Issue by Munter** [tr1s/tris-webpack-boilerplate#5]
The current use of a `.ttf` font in the boilerplate doesn't quite fit the objective of setting up a fast site. The `.ttf`-format is quite a lot heavier than `.woff2` or `.woff`.

Current browser support is good enough that the two woff-formats cover everything you need, so the TTF can be completely removed

**Comment by tr1s** [comment]
@Munter Are you implying I should find `woff` versions of my example font, or are you saying that using `woff` format automatically covers `ttf` formats even though I have a `ttf` downloaded locally?

I downloaded the font via [Google Fonts](https://fonts.google.com/specimen/Libre+Baskerville) and that's the format they gave me (`.ttf`) when I downloaded through their site.

Help me understand a bit more before I make this change.
